src/main/java/org/apache/xml/security/encryption/XMLCipherUtil.java

```diff
@@ -140,7 +140,7 @@ public static OAEPParameterSpec constructOAEPParameters(
     public static MGF1ParameterSpec constructMGF1Parameter(String mgh1AlgorithmURI) {
         LOG.log(Level.DEBUG, "Creating MGF1ParameterSpec for [{0}]", mgh1AlgorithmURI);
         if (mgh1AlgorithmURI == null || mgh1AlgorithmURI.isEmpty()) {
-            LOG.log(Level.WARNING,"MGF1 algorithm URI is null or empty. Using SHA-1 as default.");
+            LOG.log(Level.WARNING, "MGF1 algorithm URI is null or empty. Using SHA-1 as default.");
```

Comment:
Totally agree with the comment.  But this wold change the existing behavior and mya break some "useges" which is following the 
https://www.w3.org/TR/xmlenc-core1/#sec-RSA-OAEP
`The message digest function SHOULD be specified using the Algorithm attribute of the ds:DigestMethod child element of the xenc:EncryptionMethod element. If it is not specified, the default value of SHA1 is to be used. `
@coheigea if you agree I will change this to SHA-256,  and I suggest that this is duly noted in the release notes.

Reply:
I think the XML Encryption recommendation made a mistake to allow defaults for algorithms, but I don't think we need to necessarily adhere to that. That said, this could introduce interop issues if we treat a missing DigestMethod as SHA-256 instead of SHA-1. Instead, we could require the application to always specify the MGF digest method and no longer allow null to be passed in as the parameter. 

This may be best handled as a separate issue so that all APIs which have defaults are checked - for example, RSA-OAEP also defaults to SHA-1 for the digest algorithm.

Reply:
Created a ticket [SANTUARIO-618](https://issues.apache.org/jira/browse/SANTUARIO-618)